code/modules/mob/living/silicon/robot/drone/drone_manufacturer.dm

```diff
@@ -23,6 +23,21 @@
 	var/produce_drones = 1
 	var/time_last_drone = 500
 	var/drone_type = /mob/living/silicon/robot/drone
+	var/drone_ghostrole_name = "maintdrone"
+
+/obj/machinery/drone_fabricator/Initialize()
+	. = ..()
+	if(SSticker.current_state == GAME_STATE_PLAYING)
+		enable_drone_spawn()
```

Comment:
I am also not too sure about that, as this prevents admins from spawning in fabricators that arnt instantly enabled.